Dragon.cpp

```diff
@@ -1,11 +1,19 @@
 #include "Dragon.h"
 #include "DragonSlayer.h"
+#include "Character.h"
```

Comment:
Dragon.h is already including Character.h
- [ ] revise

Reply:
done.